include/bsp.h

```diff
@@ -18,7 +18,73 @@
 /*
  * TODO: Change these to real pins and ports
  */
+#define POC
 
+#ifdef POC
```

Comment:
Some pins undefined in the else case, and may need to be defined post POC. For example  X_M0_PORT is undefined